File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/non-plagiarized/09/t03.py
import math

# Get user input
weight_lbs = float(input("Enter weight in pounds: "))
feet = float(input("Enter feet: "))
inches = float(input("Enter inches: "))

# Calculate height in meters
height_m = (feet * 12 + inches) * 0.0254

# Calculate BMI
weight_kg = weight_lbs * 0.45359237
bmi = weight_kg / (height_m ** 2)

# Print BMI and classification
print(f"BMI is {bmi:.2f}")
if bmi < 18.5:
    print("underweight")
elif bmi >= 18.5 and bmi < 25:
    print("normal")
elif bmi >= 25 and bmi < 35:
    print("overweight")
else:
    print("obese")

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-03/non-plagiarized/05/T03.kt
package IR

import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    print("Enter weight in pounds: ")
    val w = sc.nextInt()
    print("Enter feet: ")
    val f = sc.nextInt()
    print("Enter inches: ")
    val i = sc.nextInt()
    val bmi = ((w * 0.45359237) / (((f * 12 + i) * 0.0254) * ((f * 12 + i) * 0.0254)))
    println("BMI is $bmi")
    when {
        bmi < 18.5 -> println("Underweight")
        bmi >= 18.5 && bmi < 25 -> println("Normal")
        bmi >= 25 && bmi < 35 -> println("Overweight")
        bmi >= 35 -> println("Obese")
    }
}

